### PR TITLE
added nested_form_for_recipe

### PR DESCRIPTION
**Pull request by OlegPavlyuk**

### CODE REVIEW

File: app/controllers/recipe_ingredients_controller.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ def new
 
   def create
     @recipe_ingredient = RecipeIngredient.new(recipe_ingredient_params)
+    redirect_to edit_recipe_path(@recipe_ingredient.recipe)
```
Comment (illia108) [inline app/controllers/recipe_ingredients_controller.rb]: maybe inside if statement?

File: app/views/ingredients/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%= form_for (@ingredient) do |f| %>
+<%= f.hidden_field :user_id %><br>
+  <div class="field">
+    <%= f.label :name%><br>
+    <%= f.text_field :name%><br>
+  </div>
+
+  <div class="field">
+    <%= f.label :calories%><br>
```
Comment (illia108) [inline app/views/ingredients/_form.html.erb]: space before %>

File: app/controllers/recipe_ingredients_controller.rb (hunk 1 above)
Comment (OlegPavlyuk) [reply app/controllers/recipe_ingredients_controller.rb]: I'll fix it,thx

File: app/views/ingredients/_form.html.erb (hunk 2 above)
Comment (OlegPavlyuk) [reply app/views/ingredients/_form.html.erb]: Ok,thx

**Pull request merged**